RT-OneEntry/ListTriggerReadThrough.cs

```diff
@@ -0,0 +1,72 @@
+﻿using Microsoft.Azure.Cosmos;
+using Microsoft.Azure.Cosmos.Linq;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Redis;
+using Microsoft.Extensions.Logging;
+using Microsoft.Identity.Client;
+using Newtonsoft.Json.Linq;
+using StackExchange.Redis;
+using System;
+using System.Collections;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.ComponentModel.Design;
+using System.Drawing;
+using System.Linq;
+using System.Reflection.Metadata;
+using System.Text;
+using System.Threading.Tasks;
+using static System.Net.Mime.MediaTypeNames;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Redis.Samples
+{
+    public record ListData
+    (
+        string id,
+        List<string> value
+    );
+    public static class ListTriggerReadThrough
```

Comment:
I would love to see docstring for all of these classes to help the reader understand what they are looking at. Ideally this should be formatted the same in every file (also the files that Phoebe and Riley have added).

Reply:
Will add this!

Reply:
Activating this until it is added.

Reply:
Following up on this- going to work with Phoebe and Riley today to make our docstrings consistent